packages/web/src/pages/upload-page/fields/AccessAndSaleField.tsx

```diff
@@ -625,7 +625,9 @@ export const AccessAndSaleMenuFields = (props: AccesAndSaleMenuFieldsProps) => {
           description={messages.hiddenSubtitle}
           disabled={noHidden}
           hintContent={
-            isScheduledRelease && isInitiallyUnlisted ? messages.hiddenHint : ''
+            isScheduledRelease && isInitiallyUnlisted !== false
```

Comment:
maybe add a comment explaining why getting rid of !== false would break it. Someone might be tempted to think it's okay

Reply:
yea good call!